components/PreviewColumn.js

```diff
@@ -19,7 +20,17 @@ export const PreviewColumn = ({ selectedSectionSlugs, getTemplate, selectedTab }
       }`}
     >
       {showPreview ? (
```

Comment:
Hi @morizkay!
According to my research this `showPreview` always it's true, so why we should keep both cases? do you know?

```js
  const showPreview = selectedTab === TAB.PREVIEW
```